script/DeployToZksync.s.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is this file still needed?

Reply:
Removed for now, working foundry plugin coming soon hopefully :)